### DISCUSSION TITLE
Expose Config Parameters as channel / entity

### DISCUSSION THREAD

**Discussion by thesebastianf** [zwave-js/zwave-js-ui#2478] [Ideas]
hey in my former solution (openhab) I added the parameters for illumination of the fibaro plugs as "channel" in command_class_configuration so I could actually control them as entity.
This allowed me to use the fibaro plug as a LED notification light in an automation. (window open = blue, alarm system on = red etc etc.)

![image](https://user-images.githubusercontent.com/13987575/174787630-9839260f-37ad-401c-8e7b-aed2ecb25dcd.png)

https://devices.zwave-js.io/?jumpTo=0x010f:0x0600:0x1000:25.25
https://devices.zwave-js.io/?jumpTo=0x010f:0x0602:0x1001:0.0

is this something which would be also possible here in zwavejs2mqtt?

![image](https://user-images.githubusercontent.com/13987575/174788177-a914944f-39bb-42cf-a14f-5cf06f06d53d.png)

**Comment by thesebastianf** [comment]
ops thats actually possible already.
sorry - how can I close this? :)

**Reply by robertsLando** [reply]
No worries it's a discussion no need to close it,you could instead explain how you can actually do it and mark it as answer :)
